### DISCUSSION TITLE
Difficulty editing the task to reduce number of trials

### DISCUSSION THREAD

**Discussion by helloshkhoo** [tatool/tatool-web#177] [Q&A]
Hi, may I know if there are updated instructions on how to edit the number of trials for a task? I'm trying to reduce the trials for Digit Memory Task. I'm trying to edit the stimuliFile and stimuliPath, using instructions about Cloudinary. However, I've been unsuccessful because of various errors (E.g., "Tatool encountered an error during the module execution. Error creating stimulus in Executable tatoolMemorySpan. No more stimuli available in current stimuliList.". I have attached the csv stimuli file I've used and the error message I face. I'll greatly appreciate any help on this! Thank you!
[digitSpan_test_short.csv](https://github.com/tatool/tatool-web/files/15502638/digitSpan_test_short.csv)
<img width="950" alt="Error" src="https://github.com/tatool/tatool-web/assets/119398962/9381ba1e-6a34-4db8-8037-8d10c5e90bc4">

**Comment by cvonbastian** [comment]
Hi helloshkhoo! I think the problem may be that you need to change the Iterations in the List that is the parent of your tatoolMemorySpan executable. This is set to 12 iterations (= trials) in the test part of the demo. Your stimuliFile having fewer trials would explain the error message "No more stimuli available in current stimuliList". Let me know if this worked! If not, can you please share your module json? 
Thanks, Claudia
